### ISSUE TITLE
Aria prop inconsistencies

### ISSUE THREAD

**Issue by mxstbr** [primer/components#494]
While writing the TypeScript definitions for all the components (#490), I noticed inconsistencies in the names of aria props.

These all set the `aria-label` attribute on the rendered element:

- `TextInput.aria-label`
- `Tooltip.text`
- `UnderlineNav.label`

It would be good if these were internally consistent with one another. I think the prop should be called `aria-label` across the board, as that's the clearest.

We could also make that change in a backwards-compatible manner, so that no existing apps built with these components are broken.

What do you think?

**Comment by emplums** [comment]
This sounds good to me!
